test/test_checks.py

```diff
@@ -1,6 +1,21 @@
+import subprocess
+
+import flake8
+
 from flake8_quotes import QuoteChecker
 import os
-from unittest import TestCase
+from unittest import expectedFailure, TestCase
+
+
+FLAKE8_2 = getattr(flake8, '__version_info__', (2, 0)) < (3, 0)
```

Comment:
Instead of sniffing the `flake8` version, maybe we should use the environment variable we already have:

```
import os

FLAKE8_VERSION = os.env.get('FLAKE8_VERSION', 2)

# ...

@expectedFailureIf(FLAKE8_VERSION == '3')
```